app/src/pages/inside/launchesPage/modals/launchAnalysisModal/launchAnalysisModal.jsx

```diff
@@ -0,0 +1,233 @@
+    id: 'launchAnalysisModal.itemOptions.investigate',
+    defaultMessage: 'To investigated items',
+  },
+  AUTO_ANALYZED: {
+    id: 'launchAnalysisModal.itemOptions.byAA',
+    defaultMessage: 'Items analyzed automatically (by AA)',
+  },
+  MANUALLY_ANALYZED: {
+    id: 'launchAnalysisModal.itemOptions.manually',
+    defaultMessage: 'Items analyzed manually',
+  },
+  VALIDATION_MESSAGE_CHOOSE_OPTION: {
+    id: 'launchAnalysisModal.validation.chooseOption',
+    defaultMessage: 'You can not perform this operation unless at least one item is chosen',
+  },
+  VALIDATION_MESSAGE_CURRENT_LAUNCH: {
+    id: 'launchAnalysisModal.validation.currentLaunch',
+    defaultMessage:
+      'You can not perform this operation for Auto-analyzed and Manually analyzed items simultaneously. Please choose one of them',
+  },
+  SUCCESS_MESSAGE: {
+    id: 'launchAnalysisModal.successMessage',
+    defaultMessage: 'Auto-analyzer has been started.',
+  },
+});
+
+@withModal('analysisLaunchModal')
+@injectIntl
+@track()
+export class LaunchAnalysisModal extends Component {
+  static propTypes = {
+    intl: intlShape.isRequired,
+    data: PropTypes.shape({
+      item: PropTypes.object.isRequired,
+      onConfirm: PropTypes.func.isRequired,
+    }),
+    tracking: PropTypes.shape({
+      trackEvent: PropTypes.func,
+      getTrackingData: PropTypes.func,
+    }).isRequired,
+  };
+
+  static defaultProps = {
+    data: {},
+  };
+  state = {
+    analyzerMode: LAUNCH_ANALYZE_TYPES.ANALYZER_MODE.LAUNCH_NAME,
+    analyzeItemsMode: [LAUNCH_ANALYZE_TYPES.ANALYZE_ITEMS_MODE.TO_INVESTIGATE],
+  };
+  onChangeRadio = (value) => {
+    this.setState({
+      analyzerMode: value,
+    });
+  };
+  onChangeCheckBox = (value) => {
+    let { analyzeItemsMode } = this.state;
+    const inState = analyzeItemsMode.includes(value);
+    if (inState) {
+      analyzeItemsMode = analyzeItemsMode.filter((item) => item !== value);
+    } else {
+      analyzeItemsMode = [...analyzeItemsMode, value];
+    }
+    this.setState({
+      analyzeItemsMode,
+    });
+  };
+  onChange = (value, type) => {
+    const { errorMessage } = this.state;
+    if (errorMessage) {
+      this.setState({
+        errorMessage: '',
+      });
+    }
+    switch (type) {
+      case 'radio':
+        return this.onChangeRadio(value);
+      default:
+        return this.onChangeCheckBox(value);
+    }
+  };
+  analysisAndClose = (closeModal) => {
+    this.props.tracking.trackEvent(LAUNCHES_MODAL_EVENTS.OK_BTN_ANALYSIS_MODAL);
+    const errorMessage = this.isInValid();
+    if (errorMessage) {
+      this.setState({
+        errorMessage,
+      });
+      return;
+    }
+    const {
+      data: {
+        item: { id },
+      },
+    } = this.props;
+    const { analyzerMode, analyzeItemsMode } = this.state;
+    const data = {
+      analyzeItemsMode,
+      analyzerMode,
+      launchId: id,
+    };
+    this.props.data.onConfirm(data);
+    closeModal();
+  };
+  isInValid = () => {
```

Comment:
I suggest to name this function `validate`.
The reason is that `is<something>` is usually used for function that return boolean